source/use-cases/local-stub.rst

```diff
@@ -30,44 +30,89 @@ test is by asking the version number.
-persistent, we also need to set the ``DNSStubListener`` to ``no`` so that is not
-changed by our router (such as with a "recommended resolver" mentioned below).
-We also want to enable the ``DNSSEC`` option so that we can verify the integrity
-the responses we get to our DNS queries. With your favourite text editor (e.g.
-:command:`nano`) we can modify the file:
+:file:`/etc/systemd/resolved.conf` in the ``DNS`` entry and uses the IP address ``127.0.0.53``.
+
+We can test this by using :command:`dig` to "example.com" and looking at the
+output.
+
+.. code-block:: bash
+
+    dig example.com
+
+Near the bottom of the output we can see ``127.0.0.53`` IP address.
+
+.. code-block:: text
+
+    ;; SERVER: 127.0.0.53#53(127.0.0.53)
+
+To change this, we are going to change the :file:`resolved.conf`.
+While just changing this file will work as long as the machine doesn't
+reboot, we need to make sure that this change is *persistent*. To do that, we
+need to change the ``DNS`` entry to be equal to ``127.0.0.1`` (or whatever IP address Unbound is bound to in your configuration) so the machine uses Unbound
+as default. So the interface would look like this in the Unbound config:
+
+.. code-block:: bash
+
+    server:
+        # specify the interface to answer queries from by ip-address.
+        interface: 127.0.0.1
+
+To test that Unbound is running, we can tell :command:`dig` to use a specific
+server with the ``@``.
+
+.. code-block:: bash
+
+    dig example.com @127.0.0.1
+
+If Unbound is running, the output should contain the address that we specified 
+in the config:
+
+.. code-block:: text
+
+    ;; SERVER: 127.0.0.1#53(127.0.0.1)
+
+If we changed :file:`resolved.conf` now, the default resolver would be persistent
+until the router wants to update it. To make sure it doesn't do that we also need to set the ``DNSStubListener`` to ``no`` so that is not changed by our
+router (such as with a "recommended resolver" mentioned below). We also want to
+enable the ``DNSSEC`` option so that we can verify the integrity the responses
+we get to our DNS queries. With your favourite text editor (e.g. :command:`nano`
+) we can modify the file:
 
 .. code-block:: bash
 
     nano /etc/systemd/resolved.conf
 
-Here, under there ``[Resolve]`` header we add (or rather, enable by removing the
-"#") the options:
+Here, under there ``[Resolve]`` header we add/substitute our changes to the
+options:
 
 .. code-block:: text
 
     [Resolve]
+    # Some examples of DNS servers which may be used for DNS= and FallbackDNS=:
+    # Cloudflare: 1.1.1.1#cloudflare-dns.com 1.0.0.1#cloudflare-dns.com 2606:4700:4700::1111#cloudflare-dns.com 2606:4700:4700::1001#cloudflare-dns.com
+    # Google:     8.8.8.8#dns.google 8.8.4.4#dns.google 2001:4860:4860::8888#dns.google 2001:4860:4860::8844#dns.google
+    # Quad9:      9.9.9.9#dns.quad9.net 149.112.112.112#dns.quad9.net 2620:fe::fe#dns.quad9.net 2620:fe::9#dns.quad9.net
```

Comment:
Not sure about listing those here; I mean endorsing. Maybe a link to https://dnsprivacy.org/public_resolvers/ and the text to use them in DNS= and FallbackDNS= is enough.